postOffice2.py

```diff
@@ -0,0 +1,148 @@
+class Message:
+    """
+    Class representing a message
+    Parameters
+    ----------
+    sender : str
+        The message sender's username.
+     message_body : str
+        The body of the message.
+    title: str
+        title of the message
+    message_id : int
+        Incremental id of the last message sent.
+
+    Attributes
+    ----------
+    sender : str
+        The message sender's username.
+    message_body : str
+        The body of the message.
+    title: str
+        title of the message
+    message_id : int
+        Incremental id of the last message sent.
+    seen : bool
+        message already seen or not
+    """
+    def __init__(self, sender, id, title, body):
+        self.id = id
+        self.title = title
+        self.body = body
+        self.sender = sender
+        self.seen = False
+
+    def __str__(self):
+        """
+        :return: return the presentation of the message
+        """
+        s = '{' + f'id: {self.id}, title: {self.title}, body: {self.body}, sender: {self.sender},' \
+                  f' seen: {self.seen}' + '}'
+        return s
+
+    def __repr__(self):
```

Comment:
__repr__ and __str__ are not the same; https://stackoverflow.com/questions/1436703/what-is-the-difference-between-str-and-repr